package.json

```diff
@@ -118,12 +118,13 @@
 		"webpack": "^5.89.0"
 	},
 	"dependencies": {
-		"@elementor/editor": "0.10.5",
-		"@elementor/editor-app-bar": "0.9.7",
+		"@elementor/editor": "0.10.6",
+		"@elementor/editor-app-bar": "0.10.1",
+		"@elementor/editor-app-bar-ui": "0.1.1",
```

Comment:
see what I've added in my PR for the editor-app-bar-ui + check that it's working
It won't work without it

Reply:
Done.